srv/salt/_modules/cephprocesses.py

```diff
@@ -12,27 +12,39 @@
 A secondary purpose is a utility to check the current state of all services.
 """
 
-def check():
+def check(roles=[]):
     """
     Query the status of running processes for each role.  Return False if any
     fail.
     """
-    processes = { 'mon': [ 'ceph-mon' ],
-                 'storage': [ 'ceph-osd' ],
-                 'mds': [ 'ceph-mds' ],
-                 'igw': [ 'lrbd' ],
-                 'rgw': [ 'radosgw' ],
-                 'ganesha': [ 'ganesha.nfsd', 'rpcbind', 'rpc.statd' ] }
+    processes = {'mon': ['ceph-mon'],
+                 'storage': ['ceph-osd'],
+                 'mds': ['ceph-mds'],
+                 'igw': ['lrbd'],
+                 'rgw': ['radosgw'],
+                 'ganesha': ['ganesha.nfsd', 'rpcbind', 'rpc.statd']}
 
     ret = True
+
+    def check_process(role):
+        for process in processes[role]:
+            result = __salt__['status.pid'](process)
```

Comment:
`result` should be avoided as a variable name. `pid` seems more appropriate here.